src/DynamoCoreWpf/ViewModels/RenderPackageFactoryViewModel.cs

```diff
@@ -49,5 +50,14 @@ public RenderPackageFactoryViewModel(IPreferences preferenceSettings)
                 TessellationParameters = { ShowEdges = preferenceSettings.ShowEdges }
             };
         }
+
+        // TODO: Remove this constructor once the IPreferences and IRenderPrecisionPreferences have been consolidated in 3.0
+        public RenderPackageFactoryViewModel(PreferenceSettings preferenceSettings)
```

Comment:
I think actually the addition of this constructor may be a breaking change - this is an overload with the same parity as another - some calls will be ambiguous I think.

Instead what about checking the type with `is` in the other constructor?